packages/imperative/src/plugins/utilities/npm-interface/install.ts

```diff
@@ -124,6 +126,11 @@ export async function install(packageLocation: string, registry: string, install
             spaces: 2
         });
 
+        iConsole.debug(`Checking for global team configuration files to update.`);
+        if (ImperativeConfig.instance.config.layers.filter((layer) => layer.global && layer.exists).length > 0) {
+            ConfigSchema.updateSchema({layer: "global"});
+        }
```

Comment:
This is the first part of https://github.com/zowe/zowe-cli/issues/1059#issuecomment-909466415. :yum: